cross/arm64/tryrun.cmake

```diff
@@ -107,7 +107,7 @@ SET( HAVE_CLOCK_MONOTONIC_COARSE_EXITCODE
      CACHE STRING "Result from TRY_RUN" FORCE)
 
 SET( HAVE_PROCFS_CTL_EXITCODE
```

Comment:
This one is no longer present configure.cmake, so it can be removed.

Reply:
Done ++